### PR TITLE
[WIP] Set FELIX_CHAININSERTMODE default to "append"

### PR DESCRIPTION
**Pull request by hakman**

### CODE REVIEW

**Comment by hakman** [comment]
@KashifSaadat do you have any thoughts on making "append" the default for `FELIX_CHAININSERTMODE` in Kops?

With the current default "insert", one of the e2e tests fails:
```
[sig-network] Services should be rejected when no endpoints exist
```

**Comment by KashifSaadat** [comment]
Hey @hakman, that suggests to me an error with the e2e tests / network policy misconfiguration.

Generally the default should always be left as `insert`, otherwise with other services potentially injecting iptables rules on a node you can run into a situation where calico chains are not hit, resulting in other issues (e.g. all traffic bypassing k8s network policies and accepted by default). I actually ran into this case in #4037.

**Comment by hakman** [comment]
Hey @KashifSaadat:). I saw some of the discussions related to that, but didn't know the whole context.
e2e tests pass with "append", but indeed may be risky.

For reference, there are a few issues related to this failing test and thought this may be an easy fix.
https://github.com/projectcalico/calico/issues/1055
https://github.com/kubernetes/kubernetes/issues/78994

**Comment by KashifSaadat** [comment]
Aha I see, thanks for the context I'll have a read through those issues to understand it better!

**Comment by hakman** [comment]
No rush. Has been like that for years :).

File: cmd/kops/create_cluster.go [hunk 1]
```diff
@@ -174,7 +174,7 @@ func (o *CreateClusterOptions) InitDefaults() {
 	o.Yes = false
 	o.Target = cloudup.TargetDirect
 	o.Models = strings.Join(cloudup.CloudupModels, ",")
-	o.Networking = "kubenet"
+	o.Networking = "calico"
```
Comment (johngmyers) [inline cmd/kops/create_cluster.go:177]: This seems to be an unrelated change.

Comment (hakman) [reply cmd/kops/create_cluster.go:177]: Yes. Pretty much the only way to test if this fixes the failing e2e test.

**Comment by hakman** [comment]
Seems that it's not worth the risk. Maybe there will be a better solution in the future.
https://github.com/kubernetes/kubernetes/issues/78994#issuecomment-611225448
